apps/daimo-mobile/src/view/screen/AddDeviceScreen.tsx

```diff
@@ -125,7 +126,11 @@ function AddDeviceScreenInner({ account }: { account: Account }) {
   return (
     <View style={ss.container.screen}>
       <ScreenHeader title="Add Device" onBack={goBack} />
-      <Spacer h={16} />
+      <Spacer h={32} />
+      <TextPara>
+        Link a new device to your account by scanning its QR code during setup.
```

Comment:
👍

Reply:
if possible put screenshot in pr

Reply:
video for passkey related screens in #337, this screen here:

<img width="256" src="https://github.com/daimo-eth/daimo/assets/6984346/fbfd7a71-43db-4aaf-b68e-ce7576162b97">